source/bayes_simulation.Rmd

```diff
@@ -327,34 +346,153 @@ Such a probability is known as a *conditional probability*.  We discuss these in
 **TODO: ADD REFERENCE TO CONDITIONAL PROBABILITY**
 -->.
 
-In our original problem, we ask what the chance is of a car being broken given that a mechanic approved it.  As discussed under "Ratios of proportions", it can be calculated with:
+In our original problem, we ask what is the chance that a car is broken given that a mechanic approved it.
+As discussed under "Ratios of proportions", it can be calculated with:
 
 $$
 P(\text{car broken | mechanic approved})
 = P(\text{car broken and mechanic approved}) / P(\text{mechanic approved})
 $$
 
-We have already used $B$ to mean "broken" (above), so let us use $A$ to mean "mechanic approved".  Then we can write the statement above in a more compact way:
+We have already used $B$ to mean "broken" (above), so let us use $A$ to mean "mechanic approved".
+Then we can write the statement above in a more compact way:
 
 $$
 P(B | A) = P(B \text{ and } A) / P(A)
 $$
 
-As we use $|$ to mean "GIVEN THAT", we can also use the $\cap$ symbol to mean "and":
+$P(B \text{ and } A)$ means the probability that $B$ and $A$ are simultaneously observed. This *joint* probability is written as
 
 $$
-P(B | A) = P(B \cap A) / P(A)
+P(B,A).
 $$
 
 To put this generally, conditional probabilities for two events $X$ and $Y$ can be written as:
 
-$P(X | Y) = P(X \cap Y) / P(Y)$
+$$
+P(X | Y) = P(X, Y) / P(Y).
+$$
 
-Where (again) $\cap$ means "and", i.e. that both happen simultaneously.
+<!---
+We need to make the equations part of the sentences, i.e. need to add '.' or ',' everywhere.
+-->
+If we rewrite this in the equivalent form:
+
+$$
+P(X, Y) = P(X | Y) P(Y).
+$$
+Since
+$$P(X, Y) = P(Y, X) = P(Y | X) P(X)
+$$
+it follows that
+
+$$
+P(X | Y) P(Y) = P(Y | X) P(X).
+$$
+This becomes the celebrated Bayes' theorem.
+
+:::{.callout-note}
+## Bayes' Theorem
+
+$$
+P(X | Y)  = \frac{P(Y | X) P(X)} {P(Y)}.
+$$
+:::
+
+It is worth thinking a moment about the meaning of $P(X)$ and $P(Y)$ in Bayes' theorem. Recall that
+
+$$
+P(X, Y) = P(X | Y) P(Y).
+$$
+If we now sum over all the values that $X$ can take, we get,
+
+$$
+\sum_X P(X, Y) = \sum_X P(X | Y) P(Y),
+$$
+or
+$$
+\sum_X P(X, Y) = P(Y) \sum_X P(X | Y).
+$$
+Since $P(X | Y)$ is a probability distribution over $X$, we have $ \sum_X P(X | Y) = 1$. We therefore have
+$$
+P(Y) = \sum_X P(X, Y).
+$$
+Similarly,
+$$
+P(X) = \sum_Y P(X, Y).
+$$
+
+These are called the marginal distributions of $X$ and $Y$.
```

Comment:
Any way to creep up more on the word "marginal" here?

Reply:
I wonder if that could do with a gentle intro page by itself? Maybe we could use that Pew democrat/republican abortion data? Happy to work on that if you both think it could be useful?

Reply:
Yes, please, Pete!